Cargo.toml

```
@@ -3,9 +3,14 @@ name = "matchspec"
 version = "0.1.0"
 edition = "2021"
 
+[lib]
+name = "rust_matchspec"
+crate-type = ["cdylib"]
+
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [dependencies]
 nom = "7"
+pyo3 = "0.7.0"
```

Updating to the latest version may help with the build error? The latest is `0.17.3`: https://github.com/PyO3/pyo3/releases

---

```suggestion
pyo3 = "0.17"
```
This is the rust equivalent of `0.17.*` if you want to do the upgrade.